lib/src/components/supa_email_auth.dart

```diff
@@ -95,6 +95,9 @@ class SupaEmailAuth extends StatefulWidget {
   /// Localization for the form
   final SupaEmailAuthLocalization localization;
 
+  /// Whether the form should display sign-in or sign-up initially
+  final bool isSigningIn;
```

Comment:
Maybe indicate that it's just the initial state?
```suggestion
  final bool isInitiallySigningIn;
```